noir-projects/aztec-nr/aztec/src/context/utils.nr

```diff
@@ -0,0 +1,3 @@
+pub fn max(a: u32, b: u32) -> u32 {
```

Comment:
I thought I saw someone else was using a generic min and or max function. Can you check if that got merged and maybe we can use it?

Reply:
Just checked and I couldn't see a "max" function in noir. I could remove this and go back to the `if` statement?